# HG changeset patch
# User Ray Donnelly <mingw.android@gmail.com>
# Date 1471185343 -3600
#      Sun Aug 14 15:35:43 2016 +0100
# Node ID d992e8de5d80dcc7638362eddc967263a78a00b8
# Parent  752045fad267219219275ae62ca4af4d1673978b
Win32: Hack sysconfig so that `CC` is set to 'gcc'

Otherwise distutils/unixccompiler.py throws an exception at:

```
    def runtime_library_dir_option(self, dir):
        ..
        compiler = os.path.basename(sysconfig.get_config_var("CC"))
```

diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -6,7 +6,9 @@
 
 from setuptools import setup, Extension
 from setuptools.command.build_ext import build_ext as _build_ext
-
+if sys.platform == "win32":
+    from distutils.sysconfig import _config_vars
+    _config_vars['CC'] = 'gcc'
 pack_name = 'rpy2'
 pack_version = __import__('rpy').__version__
 
